### PR TITLE
chore: update `meow` package to latest version

### PR DESCRIPTION
**Pull request by evilebottnawi**
> Which issue, if any, is this issue related to?

Update `meow` to latest version

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

Guys I did not forget about us, just a lot of work and I was accepted into webpack, where there is a lot of work, but I remember about stylelint and let's continue to do the best style linter :star: :star: :star:

### CODE REVIEW

**Comment by hudochenkov** [comment]
> Guys I did not forget about us, just a lot of work and I was accepted into webpack, where there is a lot of work, but I remember about stylelint and let's continue to do the best style linter ⭐️ ⭐️ ⭐️

Congratulations!

**Comment by ntwb** [comment]
Woot, congrats @evilebottnawi 🎉

File: lib/cli.js [hunk 1]
```diff
@@ -10,26 +10,85 @@ const resolveFrom = require("resolve-from");
...
+      default: false
+    },
+    syntax: {
+      alias: "s"
+    },
+    version: {
+      alias: "v",
+      type: "boolean"
+    }
+  }
```
Comment (hudochenkov) [inline lib/cli.js]: I'm not familiar with Flow, but it feels not right that this object is just a copy of `const minimistOptions`.

Maybe this one is more correct?

```
/*:: type minimistOptionsType = {
  flags: {
    "allow-empty-input": {
      alias: string,
      type: string
    },
    cache: {
      type: string
    },
    "cache-location": {
      type: string
    },
    config: {
      default: boolean,
      type: string
    },
    "config-basedir": {
      type: string
    },
    color: {
      type: string
    },
    "custom-formatter": {
      type: string
    },
    "custom-syntax": {
      type: string
    },
    "disable-default-ignores": {
      alias: string,
      type: string
    },
    fix: {
      type: string
    },
    formatter: {
      alias: string,
      type: string
    },
    help: {
      alias: string,
      type: string
    },
    "ignore-disables": {
      alias: string,
      type: string
    },
    "ignore-path": {
      alias: string
    },
    "ignore-pattern": {
      alias: string
    },
    "no-color": {
      type: string
    },
    "report-needless-disables": {
      alias: string
    },
    "stdin-filename": {
      type: string
    },
    quiet: {
      alias: string,
      type: string,
      default: boolean
    },
    syntax: {
      alias: string
    },
    version: {
      alias: string,
      type: string
    }
  }
}*/
```

Comment (evilebottnawi) [reply lib/cli.js]: @hudochenkov you are right, stupid error :smile:

**Pull request merged**